src/conditional-compilation.md

```diff
@@ -248,9 +232,7 @@ Example values:
 <!-- should we say they're active attributes here? -->
 
 The `cfg` [attribute] conditionally includes the thing it is attached to based
-on a configuration predicate.
-
-It is written as `cfg`, `(`, a configuration predicate, and finally `)`.
+on the given configuration predicate (_ConfigurationPredicate_ in the syntax above).
```

Comment:
The original is written intentionally that way, since we agreed on a style of describing the syntax in prose in most cases, and to not defer to the grammar. Can this change be reverted?